utils/flattenZoomResults.js

```diff
@@ -1,19 +1,17 @@
 function flattenZoomResults(zoomResults){
+  if (!zoomResults.length) {
+    console.log("Zoom rooms are currently empty");
+    return zoomResults;
```

Comment:
what is zoomResults sending to the client here? client is expecting an object of cohorts

Reply:
it's still sending the same stuff.  This is one function in a chain between req and res.  This just prevents the chain from throwing an error by returning an empty array instead of returning false.